tests/ext/pcntl/pcntl_fork_long_running_autoflush.phpt

```diff
@@ -51,6 +60,16 @@ function long_running_entry_point()
 
 ?>
 --EXPECTF--
+parent is enabled
+Successfully triggered flush with trace of size 3
+child is enabled
 Successfully triggered flush with trace of size 1
+Cannot run tracing closure for long_running_entry_point(); spans out of sync
```

Comment:
I'm not sure about this: forking inherently means that spans will be dropped and thus the trace end handlers skipped.

Shall we have a special marker for spans dropped by fork to silence these then?